src/helpers/utils.ts

```diff
@@ -134,3 +134,13 @@ export function n(number, format = '(0.[00]a)') {
 export function ms(number) {
   return format(number * 1e3);
 }
+
+export function calcFromSeconds(value, unit) {
```

Comment:
Its bit hard to undertsand. 

What are you trying to calculate from seconds? Days? Shouldnt it be division?

Reply:
@midgerate the unit, it is doing a division.